installation-and-configuration/office-365-connection-details.md

```diff
@@ -40,6 +40,22 @@ SysKit Trace uses certificates for two reasons.
 A certificate is created and imported into the current user personal certificates for both the user configuring SysKit Trace and the service account that is configured to run the SysKit Trace service.  
 We advise that the machine where SysKit Trace is installed is secure and not shared with other people.
 
+
+### Deleting SysKit Trace Azure AD Application from Azure
```

Comment:
i think that we can leave out the ...from Azure part. it's already evident that the app is in Azure since Azure AD is mentioned. maybe: Deleting the SysKit Trace Azure Azure AD Application. ie. here its _Create an Azure Active Directory application_ : https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal.

Reply:
Sure makes sense, I will remove it.

Reply:
And can we use Active Directory instead of AD in this heading?

Reply:
Yes I replaced all AAD with full name.